pom.xml

```diff
@@ -15,11 +15,14 @@
   <version>7.1.0-redhat-1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <properties>
-    <version.wildfly.swarm>7.1.0.redhat-33</version.wildfly.swarm>
-    <version.infinispan>8.2.8.Final</version.infinispan>
+    <version.wildfly.swarm>7.0.1.redhat-20</version.wildfly.swarm>
+    <version.infinispan>8.1.8.Final-redhat-1</version.infinispan>
```

Comment:
For some reason, after downgrading to 7.0.1, even the cute-name-service contains infinispan artifacts in the *-swarm.jar.
I had to align Infinispan version to the one used by Swarm.

I double-checked and in 7.1 it is not an issue.
Is it something worth investigating?

Reply:
Well that sounds weird...

Reply:
Could be related to version mismatches